storage/database/dynamodb.go

```diff
@@ -136,11 +137,20 @@ func NewDynamoDB(config *DynamoDBConfig, dbtype DBEntryType) (*dynamoDB, error)
 		return nil, err
 	}
 
+	dynamoOnceClient.Do(func() {
```

Comment:
Instead of using `sync.Once`, can't we just check if `dynamoDBClient` is nil?
As all `NewDynamoDB` calls are processed sequentially according to their database sequence.
I think `sync.Once` should be used when something needs to be executed only once but can be executed by different goroutines.

Reply:
review reflected :)